djoser/serializers.py

```diff
@@ -28,7 +28,7 @@ class Meta:
 class UserRegistrationSerializer(serializers.ModelSerializer):
     password = serializers.CharField(style={'input_type': 'password'},
                                      write_only=True,
-                                     validators=settings.get('PASSWORD_VALIDATORS'))
+                                     validators=utils.get_password_validators())
```

Comment:
it would be awesome if you could change the hanging indents to be more consistent with rest of the project like:

```
password = serializers.CharField(
    style={'input_type': 'password'},
    write_only=True,
    validators=utils.get_password_validators()
)
```